torchx/Makefile

```
@@ -27,8 +27,12 @@ $(TORCHX_SO): c_src/torchx.cpp c_src/nx_nif_utils.hpp
 	else \
 		ln -sf $(abspath $(LIBTORCH_DIR)/lib) $(PRIV_DIR)/libtorch ; \
 	fi
+ifneq ($(CMAKE_TOOLCHAIN_FILE),)
+	CMAKE_CONFIGURE_FLAGS=-DCMAKE_TOOLCHAIN_FILE="$(CMAKE_TOOLCHAIN_FILE)"
+endif
```

I think this should be at the top of the file, right after `C_SRC`. Also, I think this will work and be more clear:
```suggestion
ifdef $(CMAKE_TOOLCHAIN_FILE)
	CMAKE_CONFIGURE_FLAGS=-DCMAKE_TOOLCHAIN_FILE="$(CMAKE_TOOLCHAIN_FILE)"
endif
```

---

Putting CMAKE_* together indeed looks better and more clear. I'll commit these changes!